channel_output_webhook.py

```diff
@@ -16,22 +16,45 @@ class WebhookOutputChannel(OutputChannel):
 
 
     def do_send_alert(self, input_channel=None, canarydrop=None, **kwargs):
-        try:
-            payload = input_channel.format_webhook_canaryalert(
-                                          canarydrop=canarydrop,
-                                          **kwargs)
-            self.generic_webhook_send(simplejson.dumps(payload), canarydrop)
-        except Exception as e:
-            log.err(e)
+
```

Comment:
This method is looking good. There are two small changes:

1. Call the new `format_slack_canaryalert` method
2. Squash the separate exception blocks


Something like:
```
try:
    if (slack in canarydrop['alert_webhook_url']):
        payload = input_channel.format_slack_canaryalert(
                                    canarydrop=canarydrop,
                                    **kwargs)
    else:
        payload = input_channel.format_webhook_canaryalert(
                                    canarydrop=canarydrop,
                                    **kwargs)

    self.generic_webhook_send(simplejson.dumps(payload), canarydrop)
except Exception as e:
    log.err(e)
```

Reply:
fixed